region/crateria/east/West Ocean.json

```diff
@@ -2257,17 +2348,28 @@
       "name": "Bug Boost",
       "requires": [
         {"notable": "Bug Boost"},
-        "canNeutralDamageBoost",
-        "canMockball",
+        "Morph",
+        {"or": [
+          "canMockball",
+          {"and": [
+            "canInsaneJump",
+            "canBeVeryPatient"
```

Comment:
- A description of how to do it dashless would help, or strat video
- Its a little hard for Extreme but the extra bad part is losing your blue suit when you softlock